src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
```

Comment:
You should give your variables more understandable names, const inversedRobot, for example
<sub>Volodya said, he`s gonna reject our works for such names)</sub>